src/core/qgsdistancearea.cpp

```diff
@@ -384,6 +400,12 @@ QgsPointXY QgsDistanceArea::computeSpheroidProject(
   double a = mSemiMajor;
   double b = mSemiMinor;
   double f = 1 / mInvFlattening;
+  if ( ( ( a < 0 ) && ( b < 0 ) ) ||
+       ( ( p1.x() < -180.0 ) || ( p1.x() > 180.0 ) || ( p1.y() < -85.05115 ) || ( p1.y() > 85.05115 ) ) )
```

Comment:
Hmm - still seems like an artificial limitation to me. rttopo doesn't have this restriction. And we definitely have users in the poles (e.g. http://www.scar.org/data-products/quantarctica) who may be affected by this. Was this put in place to fix a particular issue, or can we change it to -90 -> +90 instead?